File: src/main/java/ravex/modules/player/FastBreak.java
package ravex.modules.player;
import ravex.modules.annotations.Module;
import ravex.modules.annotations.Parameter;
@Module(name = "FastBreak", category = "Player")
public class FastBreak {
    @Parameter(name = "Delay", min = 0, max = 4, step = 1)
    public double delay = 0;




}

File: src/main/kotlin/volthack/modules/player/FastBreak.kt
package volthack.modules.player

import volthack.setting.Category
import volthack.setting.Module

object FastBreak : Module("FastBreak", "Removes the vanilla 5-tick delay between breaking blocks legitimately to bypass anti-cheats", Category.PLAYER) {
    // FastBreak is a toggle module whose action is managed directly via MultiPlayerGameMode mixin.
}
